package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -490,7 +525,7 @@ class DumpReader(base.ReaderBase):
 
     @store_init_arguments
     def __init__(self, filename, lammps_coordinate_convention="auto",
-                 **kwargs):
+                 additional_columns=False, **kwargs):
```

Comment:
I would say `None` is more idiomatic here.

Reply:
I concur.